dev-tools/src/main/resources/plugin-metadata/plugin-descriptor.properties

```diff
@@ -58,6 +58,8 @@ jvm=${elasticsearch.plugin.jvm}
 classname=${elasticsearch.plugin.classname}
 #
 # 'java.version' version of java the code is built against
+# version string must be a sequence of nonnegative decimal integers
+# separated by "."'s and may have leading zeros
 java.version=${maven.compiler.target}
```

Comment:
We can also just mention, it should be the value of the system property java.specification.version?